velox/docs/configs.rst

```diff
@@ -179,6 +179,16 @@ sets the update mode to indicate overwriting a partition if exists.
 ``ERROR`` sets the update mode to indicate error throwing if writing
 to an existing partition.
 
+``immutable_partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``bool``
+    * **Default value:** ``false``
+
+Whether new data can be inserted into existing partitions.
```

Comment:
> Whether new data can be inserted into existing partitions.

This description is confusing since unpartitioned tables do not have partitions and this property applies only to such tables.

How about:

```
True if appending data to an existing unpartitioned table is allowed.
```

Reply:
Yes, this is much better. I have updated the comment.